src/Sulu/Bundle/AdminBundle/Controller/SecurityController.php

```diff
@@ -63,13 +53,27 @@ public function loginAction(Request $request)
      */
     public function resetAction($token)
     {
-        return $this->render(
-            'SuluAdminBundle:Security:login.html.twig',
-            [
-                'name' => $this->container->getParameter('sulu_admin.name'),
-                'locales' => $this->container->getParameter('sulu_core.locales'),
-                'token' => $token,
-            ]
-        );
+        $template = 'SuluAdminBundle:Security:login.html.dist.twig';
+        if ($this->get('kernel')->getEnvironment() === 'dev') {
+            $template = 'SuluAdminBundle:Security:login.html.twig';
+        }
```

Comment:
Would extract that into an own method, so that you can also use that in the `loginAction`.